server/views/pages/dataInsights/protectedAndResponsivityCharacteristicsTab.njk

```diff
@@ -11,7 +11,7 @@
 {% set title = "Adjudication data - Protected and responsivity characteristics" %}
 
 {% block pageTitle %}
-  {{ title }} - Adjudications - Digital Prison Services
```

Comment:
Don't think we want to update this - it would revert https://github.com/ministryofjustice/hmpps-manage-adjudications/pull/789